src/settings/apiSettings.ts

```diff
@@ -0,0 +1,7 @@
+// constants for the collections, just in case some get renamed (e.g. venues->spaces)
+export const COLLECTION_SPACES = "venues";
+export const COLLECTION_WORLDS = "worlds";
```

Comment:
[NB] In my opinion, it would be better if these are in another file.

Reply:
Maybe, couldn't figure our where exactly, so this was the most visible place. Just a preparation for the time "rename venues to spaces in the DB" comes to be worked on.